src/test/resources/superclass/expected/com/example/Simple.java

```diff
@@ -0,0 +1,15 @@
+package com.example;
+
+class BigSimple {
+    public void printMessage() {
+        throw new Error();
+    }
+}
+
+public class Simple extends BigSimple {
+    @Override
+    public void printMessage() {
+        super.printMessage();
+        System.out.println("It's me");
```

Comment:
This expected output isn't correct (or, at least, isn't a great test), because this won't compile without an implementation for `System.out.println`. Since that's in the JDK, it's probably okay not to print anything for now, but it would be better to avoid having these kind of complications if we can avoid it.